README.adoc

```diff
@@ -177,43 +182,43 @@ include::finish/src/main/java/io/openliberty/guides/data/Packages.java[]
 
 [role="code_command hotspot file=0", subs="quotes"]
 ----
-#Update the `Packages.java` class.#
+#Update the `Packages.java` class:#
 `src/main/java/io/openliberty/guides/data/Packages.java`
 ----
 
 === Sort
-In addition to the `@OrderBy` annotation it is possible to provide sorting at runtime. This is accomplished by adding a `Sort` as a parameter to your query method.
+In addition to the `@OrderBy` annotation it is possible to provide sorting at runtime. This is accomplished by adding a `Sort` object as a parameter to your query method:
```

Comment:
```suggestion
In addition to the `@OrderBy` annotation it is possible to provide sorting at runtime. This is accomplished by adding a `Sort` parameter to your query method:
```
An object generally refers to an instance, so I'm not sure it makes sense in a method declaration, but maybe this is clearer? Hopefully at a minimum the following code example clarifies it.